File: zkext/trunk/src/web/js/zkext/ui/form/Text.js
/* Text.js*/

zkext.ui.form.Text = zk.$extends(zkext.ui.form.Field,{	 
	$define: {	 
		 allowBlank:function(val){
			 this.setProperty("allowBlank",null,val);
		 }
	},
	
	configure_:function(){	
		this.$supers('configure_');		
							 
	},
	
	createExt_:function(){
		this.newInstance('Ext.form.field.Text');
	} 	
});

File: zkext/trunk/src/com/ff/ui/form/Text.java
package com.ff.ui.form;

import com.ff.annotation.Property;

public class Text extends Field{

	@Property
	private Boolean allowBlank;

	public Boolean getAllowBlank() {
		return allowBlank;
	}

	public void setAllowBlank(Boolean allowBlank) {
		this.allowBlank = allowBlank;
	}
	
	
}
